tests/rptest/tests/fetch_fairness_test.py

```diff
@@ -89,15 +90,26 @@ def multiple_partitions(count):
         # configure kcl to fetch at most 1 byte in single fetch request,
         # this way we should receive exactly one record per each partition
 
+        consumed_frequency = defaultdict(lambda: 0)
         kcl = KCL(self.redpanda)
+
+        expected_frequency = 5
```

Comment:
Would be good to define this relative to records_per_partition (e.g. as records_per_partition/2) to make the relationship clearer.

Reply:
Noah suggested in the issue to increase the number of records to get better statistical properties, I think that's a good idea too (e.g. set records per topic to something like 100 rather than 10)